tests/unit/mock-data.json

```diff
@@ -35,6 +35,96 @@
       "name": "Hedwig",
       "owner": "Harry Potter",
       "species": "Owl"
+    },
+    {
+      "id": "8",
+      "name": "pip (deceased)",
```

Comment:
😢 (are you serious you named it pip?)

Reply:
I did, he was a great fish :(
![IMG_20170206_212009](https://user-images.githubusercontent.com/39310585/62577677-4cfe2b00-b854-11e9-89af-654616afbede.jpg)

Reply:
lol, his photo is in part of our API history now.